client/components/data/query-jetpack-onboarding-settings/README.md

```diff
@@ -0,0 +1,44 @@
+Query Jetpack Onboarding Settings
+=================================
+
+`<QueryJetpackOnboardingSettings />` is a React component used in managing network requests for Jetpack Onboarding Settings.
+
+## Usage
+
+Render the component, passing `siteId`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import QueryJetpackOnboardingSettings from 'components/data/query-jetpack-onboarding-settings';
+import { getJetpackOnboardingSettings } from 'state/selectors';
```

Comment:
Nit of the nits: 🔤

Reply:
Okay, I'll move `{ getJetpackOnboardingSettings }` before `QueryJetpackOnboardingSettings` -- but no further up, because external vs internal, k?